README.md

```diff
@@ -666,100 +795,161 @@ Specifies the addresses the server accepts connections to. Valid values:
 
 ##### `locale`
 
-Sets the default database locale for all databases created with this module. On certain operating systems this is used during the `template1` initialization as well, so it becomes a default outside of the module. Default: undef, which is effectively `C`. **On Debian, you must ensure that the 'locales-all' package is installed for full functionality of PostgreSQL.**
+Sets the default database locale for all databases created with this module. On certain operating systems this is used during the `template1` initialization as well, so it becomes a default outside of the module.
+
+Default value: `undef`, which is effectively 'C'.
+
+**On Debian, you must ensure that the 'locales-all' package is installed for full functionality of PostgreSQL.**
 
 ##### `log_line_prefix`
 
-Set a prefix for the server logs. Default: `'%t '`
+Set a prefix for the server logs.
+
+Default value: '%t'
 
 ##### `manage_pg_hba_conf`
 
-Whether to manage the pg_hba.conf. If set to true, Puppet overwrites this file. If set to false, Puppet does not modify the file. Valid values: true, false. Default
+Whether to manage the `pg_hba.conf`.
+
+If set to `true`, Puppet overwrites this file.
+
+If set to `false`, Puppet does not modify the file.
+
+Valid values: `true`, `false`.
+
+Default value: TODO
```

Comment:
Defaults to the value in globals set in `postgresql::globals::manage_pg_hba_conf` (which defaults to `true`).